Framework/include/QualityControl/HistoMerger.h

```diff
@@ -0,0 +1,65 @@
+///
+/// \file   HistoMerger.h
+/// \author Piotr Konopka
+///
+
+#ifndef QUALITYCONTROL_HISTOMERGER_H
+#define QUALITYCONTROL_HISTOMERGER_H
+
+#include <vector>
+#include <memory>
+
+#include <TH1.h>
+#include <Framework/Task.h>
+#include <Headers/DataHeader.h>
+#include "Common/Timer.h"
+
+#include "QualityControl/MonitorObject.h"
+
+namespace o2 {
+namespace quality_control {
+namespace core {
+
+
+/// \brief A crude histogram merger for development purposes.
+class HistoMerger : public framework::Task {
+public:
+  /// Constructor
+  HistoMerger(std::string mergerName);
+
+  /// Destructor
+  ~HistoMerger() override;
+
+  /// \brief Checker init callback
+  void init(framework::InitContext& ctx) override;
+
+  /// \brief Checker process callback
+  void run(framework::ProcessingContext& ctx) override;
+
+  void configureEdges(
```

Comment:
I am not sure about this name. Not very important though. "configureInputsOutputs" ? "configureTransputs" ? The latter is a new word in my vocabulary :)

Reply:
'Edges' because of the graph theory. I am fine with `configureIO` or `configureInputsOutputs` though